transmart-api-server/grails-app/controllers/org/transmartproject/api/server/ApiServerUrlMappings.groovy

```diff
@@ -0,0 +1,10 @@
+package org.transmartproject.api.server
+
+class ApiServerUrlMappings {
+
+    static mappings = {
+        "/"(redirect: "/open-api/index.html")
```

Comment:
would it always redirect independently what `Accept` value specified in the header?